src/components/Card/Card.css

```diff
@@ -67,12 +68,15 @@
 	font-weight: 100;
 	padding:10px 24px 18px 24px;
 	margin:0;
-	line-height: 1.6;
+	line-height: 1.3;
 	letter-spacing: 0.17px;
 }
 @media only screen and (max-width : 359px),
 only screen and (max-device-width : 359px){
 	.card{
-		transform: scale(0.9);
+		transform: scale(0.85);
+		position: relative;
+		left: -6.25vw;
+		margin-bottom: 0;
 	}
 }
```

Comment:
Add a new line here. You may configure your text editor as suggested by @itaditya in #1